### ISSUE TITLE
SASS hot-reloading on with-external-scoped-css example

### ISSUE THREAD

**Issue by JoseMartinC** [zeit/next.js#2495]
- [x] I have searched the [issues](https://github.com/zeit/next.js/issues) of this repository and believe that this is not a duplicate.

Hello guys! 

First things first: thanks a lot to the contributors for the framework : )

I'm contacting because I'm having hard times trying to hot-reload the .sass on this example: https://github.com/zeit/next.js/tree/v3-beta/examples/with-external-scoped-css

My goal is to have a folder per component / page, with a hot-reloaded js and scss file, using scoped css, and without adding `<style dangerouslySetInnerHTML />` on every component or without using template string kind of css technology as style-jsx or styled-components.

```
components
├── comp1
│   ├── index.js
│   └── styles.scss
├── comp2
│   ├── index.js
│   └── styles.scss
└── comp3
    └── index.js
    └── styles.scss
```

So the thing is that there's a related PR mentioned in the example as a potential fix for the hot-reloading ( https://github.com/zeit/next.js/pull/823), and I was wondering if anybody has any example / tip on how the functionality added by the PR could be used to customize the hot-reloading so the changes on the sass are also detected and injected.

I have tried and tested a lot without success, including the approached on [this thread](https://github.com/zeit/next.js/issues/544), but none is fitting completely what I want to achieve, so any help would be really appreciated.

Thanks a lot in advance!!

**Comment by timneutkens** [comment]
> So the thing is that there's a related PR mentioned in the example as a potential fix for the hot-reloading ( [#823](https://github.com/zeit/next.js/pull/823)), and I was wondering if anybody has any example / tip on how the functionality added by the PR could be used to customize the hot-reloading so the changes on the sass are also detected and injected.

this actually doesn't fix the issue you're having 😢

**Comment by timneutkens** [comment]
Lets track this in #1615 and https://github.com/zeit/next.js/issues/1245
